bunnyhop/billing.py

```diff
@@ -0,0 +1,16 @@
+from bunnyhop.base import BaseBunny
+
+
+class Billing(BaseBunny):
+    
+    def get(self):
+        api_url = self._URL + '/billing'
+        header = self._GetHeaders()
+        response = self._CallApi(api_url, "GET", header)
+        return self._FormatResponse(response)
```

Comment:
self.format_response should be part of the self.call_api method
Also, you can just call 
```python
self.call_api(f"{self.endpoint_url}/billing", "GET", self.get_headers())
```